iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -261,6 +268,13 @@ public boolean nextKeyValue() throws IOException {
       while (true) {
         if (currentIterator.hasNext()) {
           current = currentIterator.next();
+          Configuration conf = context.getConfiguration();
```

Comment:
We can set it as an object attribute instead of getting it again and again

Reply:
Sure